src/unstract/sdk/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.15.1"
+__version__ = "0.16.0"
```

Comment:
If it a not a breking changes in perspective of SDK ,  0.15.2 is enough. It's only static file. But need to ensure it will not break.

Reply:
Agreed, this might not be a breaking change in the context of the SDK - but do check the release notes of adapters and confirm

Reply:
done